src/components/Filmteam/FilmTeam.scss

```diff
@@ -2,36 +2,47 @@
 @import '../../styles/mixins';
 
 .film-team-container {
-  height: 45vh;
-  width: 100%;
-  padding: 2vh;
+  grid-column-start: 1;
+  grid-column-end: 5;
+  background-color: whitesmoke;
```

Comment:
As we think about the layout of the page, we may end up needing to put this `whitesmoke` on opposite components.